src/plugins/discover/public/application/components/default_discover_table/table_header_column.tsx

```diff
@@ -169,7 +169,12 @@ export function TableHeaderColumn({
 
   return (
     <th data-test-subj="docTableHeaderField" className="docTableHeaderField">
-      <span data-test-subj={`docTableHeader-${name}`}>
+      <span
```

Comment:
We should give the `th` [a role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/columnheader_role) and move the `aria-label` to it.